src/libs/antares/study/cleaner/cleaner-v20.cpp

```diff
@@ -195,6 +195,8 @@ static void PreflightVersion20_area(PathList& e, PathList& p, const Area* area,
     e.add(buffer);
     buffer.clear() << "input/areas/" << id << "/optimization.ini";
     e.add(buffer);
+    buffer.clear() << "input/areas/" << id << "/adequacy_patch.ini";
```

Comment:
Well done ! This tool us easily forgotten. Adding this here should prevent the study cleaner from removing these newly introduced files.